strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-pypi-layout-provider/src/main/java/org/carlspring/strongbox/artifact/metadata/PypiArtifactMetadata.java

```diff
@@ -0,0 +1,242 @@
+package org.carlspring.strongbox.artifact.metadata;
```

Comment:
@carlspring can you please advice here? 

Currently we have following metadata package namings:
* `org.carlspring.strongbox.storage.metadata.nuget` for Nuget
* `org.carlspring.strongbox.npm.metadata` for Npm
* `org.carlspring.strongbox.artifact.metadata` here for Pypi

we better have same naming for all metadata related packages. As for me the `org.carlspring.strongbox.artifact.metadata` looks good (or maybe `org.carlspring.strongbox.artifact.metadata.npm`) but in this case I suggest to create a task to rename Nuget and Npm metadata packages same way.

Reply:
Hi,

I think that we should have all layout providers as sub-packages under `org.carlspring.strongbox.storage.metadata`, such as, for example, `org.carlspring.strongbox.storage.metadata.nuget`.

Reply:
ok, thanks for clarifying @carlspring 
@anki2189 can you please rename package to `org.carl spring.strongbox.storage.metadata.pypi`?

Reply:
Should I move this class to storage-api module ?

Reply:
I think you should do it same way as for `org.carlspring.strongbox.storage.metadata.nuget`

Reply:
> Should I move this class to storage-api module ?

No, keep them in their respective modules, just rename the package.